workspaces/rbac/plugins/rbac/src/components/ConditionalAccess/ComplexConditionRowButtons.tsx

```diff
@@ -74,15 +72,23 @@ export const ComplexConditionRowButtons = ({
     (criteria === criterias.allOf || criteria === criterias.anyOf) && (
       <Box mt={1} mb={1}>
         <Button
-          className={classes.addRuleButton}
+          sx={{
+            display: 'flex !important',
+            color: theme => theme.palette.primary.light,
+            textTransform: 'none',
+          }}
           size="small"
           onClick={handleAddRule}
         >
           <AddIcon fontSize="small" />
           Add rule
         </Button>
```

Comment:
I know, this was mostly the same code as before. But instead of custom styles the Button supports this props as well. Can you check if this looks the same:

```tsx
<Button color="primary" size="small" startIcon={<Add Icon fontSize="small" />}>
  Add rule
</Button>
```

If primary color light and primary color isn't the same here, we can maybe anyway use startIcon?